ed448-goldilocks/src/edwards/affine.rs

```diff
@@ -69,36 +69,36 @@ impl AffinePoint {
         y: FieldElement::ONE,
     };
 
-    pub(crate) fn isogeny(&self) -> Self {
-        let x = self.x;
-        let y = self.y;
-        let mut t0 = x.square(); // x^2
-        let t1 = t0 + FieldElement::ONE; // x^2+1
-        t0 -= FieldElement::ONE; // x^2-1
-        let mut t2 = y.square(); // y^2
-        t2 = t2.double(); // 2y^2
-        let t3 = x.double(); // 2x
-
-        let mut t4 = t0 * y; // y(x^2-1)
-        t4 = t4.double(); // 2y(x^2-1)
-        let xNum = t4.double(); // xNum = 4y(x^2-1)
-
-        let mut t5 = t0.square(); // x^4-2x^2+1
-        t4 = t5 + t2; // x^4-2x^2+1+2y^2
-        let xDen = t4 + t2; // xDen = x^4-2x^2+1+4y^2
-
-        t5 *= x; // x^5-2x^3+x
-        t4 = t2 * t3; // 4xy^2
-        let yNum = t4 - t5; // yNum = -(x^5-2x^3+x-4xy^2)
-
-        t4 = t1 * t2; // 2x^2y^2+2y^2
-        let yDen = t5 - t4; // yDen = x^5-2x^3+x-2x^2y^2-2y^2
-
-        Self {
-            x: xNum * xDen.invert(),
-            y: yNum * yDen.invert(),
-        }
-    }
+    // pub(crate) fn isogeny(&self) -> Self {
+    //     let x = self.x;
+    //     let y = self.y;
+    //     let mut t0 = x.square(); // x^2
+    //     let t1 = t0 + FieldElement::ONE; // x^2+1
+    //     t0 -= FieldElement::ONE; // x^2-1
+    //     let mut t2 = y.square(); // y^2
+    //     t2 = t2.double(); // 2y^2
+    //     let t3 = x.double(); // 2x
+
+    //     let mut t4 = t0 * y; // y(x^2-1)
+    //     t4 = t4.double(); // 2y(x^2-1)
+    //     let xNum = t4.double(); // xNum = 4y(x^2-1)
+
+    //     let mut t5 = t0.square(); // x^4-2x^2+1
+    //     t4 = t5 + t2; // x^4-2x^2+1+2y^2
+    //     let xDen = t4 + t2; // xDen = x^4-2x^2+1+4y^2
+
+    //     t5 *= x; // x^5-2x^3+x
+    //     t4 = t2 * t3; // 4xy^2
+    //     let yNum = t4 - t5; // yNum = -(x^5-2x^3+x-4xy^2)
+
+    //     t4 = t1 * t2; // 2x^2y^2+2y^2
+    //     let yDen = t5 - t4; // yDen = x^5-2x^3+x-2x^2y^2-2y^2
+
+    //     Self {
+    //         x: xNum * xDen.invert(),
+    //         y: yNum * yDen.invert(),
+    //     }
+    // }
```

Comment:
Can you just remove this?

Getting this PR reviewed seems doable, but #1306 is quite large and might take awhile

Reply:
Sure!

Reply:
Done.